gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -23,4 +23,92 @@ servers:
         default: http
         description: The schema of the Zeebe REST API server
 
-paths: {}
+paths:
+  /topology:
+    get:
+      summary: Get cluster topology.
+      description: Obtains the current topology of the cluster the gateway is part of.
+      responses:
+        '200':
+          $ref: '#/components/responses/TopologyResponse'
+
+components:
+  responses:
+    TopologyResponse:
+      description: Failed to accept request. No user task was found.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/TopologyResponse'
+
+  schemas:
+    TopologyResponse:
+      description: The response of a topology request.
+      type: object
+      properties:
+        brokers:
+          description: List of brokers that are part of this cluster
+          type: array
+          nullable: true
+          items:
+            $ref: "#/components/schemas/BrokerInfo"
+        clusterSize:
+          description: The number of nodes in the cluster
+          type: integer
```

Comment:
:question: Quick question, how does OpenAPI treat `integer` and `nullable`? Will it set it to 0? Will it be boxed (e.g. `Integer`) which can be nulled?

Reply:
What does it mean concretely? That it will get a zero-value or that it will actually generate a model with a nullable value (e.g. using `Integer` instead of a primitive)?

Reply:
I'll also add the `format: int32` property for any `Integer` properties in the OpenAPI spec that we define.

For `Long` properties, we will need to define `format: int64`.